x-pack/plugins/security/public/views/management/edit_role/lib/get_application_privileges.js

```diff
@@ -3,23 +3,15 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-export function getKibanaPrivilegesViewModel(applicationPrivileges, roleKibanaPrivileges) {
-  const viewModel = applicationPrivileges.reduce((acc, applicationPrivilege) => {
-    acc[applicationPrivilege.name] = false;
-    return acc;
-  }, {});
-
-  if (!roleKibanaPrivileges || roleKibanaPrivileges.length === 0) {
-    return viewModel;
-  }
 
-  const assignedPrivileges = roleKibanaPrivileges.global;
-  assignedPrivileges.forEach(assignedPrivilege => {
-    // we don't want to display privileges that aren't in our expected list of privileges
-    if (assignedPrivilege in viewModel) {
-      viewModel[assignedPrivilege] = true;
+export function getKibanaPrivilegesViewModel(applicationPrivileges, roleKibanaPrivileges) {
```

Comment:
Yay! It seems like we don't really need this anymore now that we aren't rendering the "checkbox list" you think we can remove it?